browser-extension/src/popup/popup.js

```diff
@@ -0,0 +1,26 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const form = document.getElementById('attestationForm');
+  const generateProofButton = document.getElementById('generateProof');
+
+  chrome.storage.sync.get(['age', 'signature', 'issuerPublicKey'], (result) => {
+    document.getElementById('age').value = result.age || '';
+    document.getElementById('signature').value = result.signature || '';
+    document.getElementById('issuerPublicKey').value =
+      result.issuerPublicKey || '';
+  });
+
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const age = document.getElementById('age').value;
+    const signature = document.getElementById('signature').value;
+    const issuerPublicKey = document.getElementById('issuerPublicKey').value;
+
+    chrome.storage.sync.set({ age, signature, issuerPublicKey }, () => {
```

Comment:
when looking at this code I noticed that eslint complains because it doesn't know `chrome`. I think there's an annotation you can add at the top of a file to specify some variable as an existing global -- could you add that?

also, can you install https://www.npmjs.com/package/@types/chrome so that we get better intellisense feedback about `chrome`?

Reply:
I think it was this annotation:
```ts
/* global chrome */
```